### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'mask')

### ISSUE THREAD

**Issue by krajcikondra** [iamcal/js-emoji#153]
Hi,

am facing to following issue:

```
    import EmojiConvertor from 'emoji-js';

    const emojiConvert = (sanitizedHTML: string, emojiSet?: string): string => {
        const emojiConvertor = new EmojiConvertor();
        const msgHtml = sanitizedHTML.replace(/\n$/,'');
        emojiConvertor.img_sets.google.sheet = '/emoji-data/sheet_' + emojiSet + '_64.png';
        emojiConvertor.use_sheet = false;

        emojiConvertor.replace_mode = 'img';
        emojiConvertor.img_set = emojiSet;
        emojiConvertor.include_title = true;

        return emojiConvertor.replace_colons(msgHtml);
    }
```

And when I call function 

```
emojiConvert('hello :alien:');
```

I get following error:
TypeError: Cannot read properties of undefined (reading 'mask')

Screen here: https://ctrlv.cz/gpcx

I am using emoji-js version 3.6.0

Some tips?

**Comment by iamcal** [comment]
I can't get as far as you to check, because I don't know what toolchain and VM you're using (this is typescript, so how are you compiling it and where are you running it?).

My _guess_ would be that you're passing an invalid value to `.img_set` and so it's failing to find all the information it needs to initialize. What happens if you call `emojiConvert('hello :alien:', 'apple');` instead?

**Comment by krajcikondra** [comment]
I am sorry, my mistake. I passed emojiSet undefined.
@iamcal thanks

**Comment by iamcal** [comment]
The errors should be less confusing!

I've just landed PR #156 on master which in future will throw an exception in this situation, explaining what is wrong.
